packages/beta/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 - add sorting for alerts ([#1097](https://github.com/cognitedata/cognite-sdk-js/issues/1097)) ([1a08cca](https://github.com/cognitedata/cognite-sdk-js/commit/1a08ccaf7a2d0d16e057bcd8819774c3b5a513dd))
 
+- **files:** add support for multi part file upload ([#1088] closes (https://github.com/cognitedata/cognite-sdk-js/issues/1088))
+
```

Comment:
please don't touch the changelog, it's autogenerated from commit messages

Reply:
👍🏽